index.html

```diff
@@ -845,15 +845,17 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 <li>Increase <a>resource timing buffer current size</a> by 1.</li>
 </ol>
 </li>
-<li>Otherwise, if the <a>resource timing buffer full flag</a> is
-false, run the following substeps:
+<li>If <a>resource timing buffer current size</a> is greater than
+or equal to <a>resource timing buffer size limit</a> and the
+<a>resource timing buffer full flag</a> is false, run the following
+substeps:
 <ol style='list-style-type: lower-latin;' data-link-for=
 "Performance">
 <li>Set the <a>resource timing buffer full flag</a> to true.</li>
 <li><a data-cite="!DOM/#concept-event-fire">Fire a simple event</a>
```

Comment:
I removed the word 'simple' because I didn't see it but I also didn't find any recent changes (< 1year) to the fire event algorithm. Is there anything in particular you were thinking about?

Reply:
Apologies, I was recalling in Payment Request we had to formulate the prose for firing the event in a very particular way, but that's because we defined a new event interface. With just using `Event`, I think your current text is all good.